### ISSUE TITLE
adding extra babel config replaces the babel config instead of merging them

### ISSUE THREAD

**Issue by NK-WebDev** [ricokahler/sanity-codegen#58]
In our sanity studio project we are using classProperties like this

```
 static propTypes = {
    type: PropTypes.shape({
      title: PropTypes.string,
    }).isRequired,
    value: PropTypes.string,
    onChange: PropTypes.func.isRequired,
  };
```

so we are using  the @babel/plugin-proposal-class-properties plugin to enable this but since the default babel config that comes with the cli doesn't have this plugin we need to add this, is there a way of merging the babel config with the default config? right now if we add our babel config to the babelOptions, it replaces the default with our config which doesn't have the shim for the sanity-parts which breaks the cli.

Do you have any suggestions on how to handle this?

**Comment by ricokahler** [comment]
to handle this one, i'll just add that to the default babel config. very very common. give me 5 mins

**Comment by ricokahler** [comment]
I just released v0.8.1.

This version includes those extra babel plugins. Additionally, I exported the default babel config if you wanted to use that for merging.

Let me know if that does some for you!

https://github.com/ricokahler/sanity-codegen/pull/60/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R56-R57

**Comment by NK-WebDev** [comment]
Thanks for the quick fix
